src/include/utils/xml.h

```diff
@@ -19,6 +19,9 @@
 #include "fmgr.h"
 #include "nodes/execnodes.h"
 #include "nodes/primnodes.h"
+#ifdef USE_LIBXML
+#include <libxml/tree.h>
```

Comment:
This is needed for what? And I think header inclusion does not need be conditional

Reply:
This is needed for what? 
* Header `<libxml/tree.h>` contains definition of `xmlNodePtr` which is used in xml_parse_wrapper declaration. 
And I think header inclusion does not need be conditional
* This header will not be available if PostgreSQL is not compiled with `--with-libxml` flag or libxml2 related dependencies are not installed. Hence to avoid build failure in such cases we need this condition. Similar condition is added at all the places where this header is imported in PostgreSQL.